client/me/purchases/cancel-purchase/refund-information.jsx

```diff
@@ -43,6 +46,67 @@ const CancelPurchaseRefundInformation = ( {
 		} );
 	};
 
+	const featuresList = ( productSlug, hasDomain ) => {
+		if ( typeof productSlug !== 'string' ) {
+			return null;
+		}
+
+		const planCancellationFeatures = getPlanCancellationFeatures( productSlug, hasDomain );
+
+		return (
+			<Fragment>
+				<div className="cancel-purchase__refund-information-description-header">
+					{ i18n.translate( 'When your %(productName)s plan expires you will loose access to:', {
+						args: {
+							productName: getName( purchase ),
+						},
+					} ) }
+				</div>
+				<ul className="cancel-purchase__refund-information--list-plan-features">
+					{ planCancellationFeatures.featureList.map( ( cancellationFeature ) => {
+						return (
+							<li key={ cancellationFeature }>
+								<Gridicon
+									className="cancel-purchase__refund-information--item-cross-small"
+									size={ 24 }
+									icon="cross-small"
+								/>
+								{ getCancellationFeatureByKey( cancellationFeature ) }
+							</li>
+						);
+					} ) }
+					{ planCancellationFeatures.andMore && (
+						<li
+							className="cancel-purchase__refund-information--item-more"
+							key="cancellationAndMore"
+						>
+							<span className="cancel-purchase__refund-information--item-more-span">
+								{ i18n.translate( 'And more' ) }
+							</span>
+						</li>
+					) }
+				</ul>
+			</Fragment>
+		);
+	};
+
+	const cancelSubscriptionDescription = () => {
+		const planSlug = site.plan.product_slug;
+
+		return (
+			<div>
+				<p className="cancel-purchase__refund-information-subtitle">
+					{ i18n.translate(
+						'If you cancel your plan subscription your site may appear broken and things may not work properly.'
```

Comment:
"things" does not sound very professional here, and the whole sentence is kind of wishy-washy.... In general I feel like we reserve this kind of language for cases where we know something is likely to undergo major changes (site will lose its current premium theme, Atomic site will be reverted, etc).

Could maybe say something simpler like "your site will lose functionality and its appearance may change", but overall -- does this sentence even add anything useful, given the more specific list of things that will be lost which is displayed right underneath it?

Reply:
I updated the copy, LMK if you want me to remove this text

Reply:
I'm not sure what others think, but personally that's what I would do.